src/JBrowse/Digest/Crc32.js

```diff
@@ -22,6 +22,26 @@ var Crc32 = {
         return re;
     },
 
+    crc32_raw: function( bytes, crc ) {
+        if( crc == window.undefined ) crc = 0;
```

Comment:
you want a `===` here i think.

Reply:
I changed this to just use function default parameters https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters